recipes/apstools/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "apstools" %}
+{% set version = "1.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apstools-{{ version }}.tar.gz
+  sha256: 459747b504df5def3564bf6e4d990e8b7eff42910dbeeb1dd73f9a76c9960649
+
+build:
+  entry_points:
+    - bluesky_snapshot = apstools.snapshot.application:snapshot_cli
+    - bluesky_snapshot_viewer = apstools.snapshot.application:snapshot_gui
+    - spec2ophyd = apstools.migration.spec2ophyd:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py>=36, py<310]
```

Comment:
This one can probably be noarch. Also, I'm not sure a comma works there as you expected. We have `and` or, `or`.

Reply:
Not seeing the comma here.  Where is it?

Reply:
`py>=36 and py<310`

Reply:
Thanks!

Reply:
If you are going the noarch route you need to specify the python constraint on `host` and `run`, but please check if there are no platform specific code and/or scripts in the package.

Reply:
Thought I was following your comment about:

> This one can probably be noarch. 

Did not use `noarch:python` since there is a bash shell script.

Reply:
Then you need to remove this line: https://github.com/conda-forge/staged-recipes/pull/19839/files#diff-060b85fd13f9c3afc4d4646e58643f7af978f600086d93b19c828884927372e0R17